packages/runtime/src/query.ts

```diff
@@ -1,27 +1,19 @@
 import type {
+  AnyValueFilter,
   BaseQuery,
   CeramicApi,
+  ObjectFilter,
   Page,
   Pagination,
   PaginationQuery,
+  QueryFilters,
   StreamState,
 } from '@ceramicnetwork/common'
 import { ModelInstanceDocument } from '@ceramicnetwork/stream-model-instance'
 import type { Connection, ConnectionArguments } from 'graphql-relay'
 
 export type ConnectionQuery = BaseQuery & ConnectionArguments
 
-export function toBaseQuery(source: BaseQuery): BaseQuery {
```

Comment:
I guess the point of this function was to strip out any extra params that aren't expected?  Is there a reason why you've removed it now?  What changed that this is no longer the desired behavior?

Reply:
I think this is now covered by https://github.com/ceramicstudio/js-composedb/pull/144/files#diff-e080742723a30fbf28f48db65ef61685cc18bf45ef6de97ed5df6ce7b20597acR181 where we pass query filters. 

Tagging @PaulLeCam so he can verify.